server/src-lib/Hasura/SQL/Types.hs

```diff
@@ -378,6 +382,7 @@ pgTypeOid PGJSONB       = PTI.jsonb
 -- we are using the ST_GeomFromGeoJSON($i) instead of $i
 pgTypeOid PGGeometry    = PTI.text
 pgTypeOid PGGeography   = PTI.text
+pgTypeOid PGRaster      = PTI.auto
```

Comment:
Should we use [`RT_ST_RastFromHexWKB`](https://postgis.net/docs/RT_ST_RastFromHexWKB.html) or [`RT_ST_RastFromWKB`](https://postgis.net/docs/RT_ST_RastFromWKB.html) for raster literals, the same way we use `ST_GeomFromGeoJSON` for geometry types?